src/vuex/modules/wallet.js

```diff
@@ -18,7 +18,7 @@ async function createPolkadotAddress(seedPhrase, addressPrefix) {
 
   const keyring = new Keyring({
     ss58Format: Number(addressPrefix),
-    type: "ed25519"
+    type: "sr25519"
```

Comment:
lol so we where using ed25519 and need to switch to schnorr?

Reply:
true! ed25519 is only needed if you want to use your account for a validator role.